src/imports.rs

```diff
@@ -40,6 +40,12 @@ pub struct ProjectImport {
     pub line_no: u32,
 }
 
+pub struct Dependency {
+    pub file_path: PathBuf,
+    pub absolute_path: PathBuf,
+    pub import: ProjectImport,
+}
```

Comment:
nit: I'd expect this to stay as a private struct in reports.rs since all its construction and usage is in that file

Reply:
Ah whoops, moved it out as I was originally using it in `cli.rs` but then changed that - will move back